profile/CSIP.xml

```diff
@@ -350,7 +350,7 @@
             <requirement ID="CSIP20" REQLEVEL="SHOULD" RELATEDMAT="VocabularyStatus" EXAMPLES="dmdSecExample1">
                 <description>
                     <head>Status of the descriptive metadata</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">Status of the metadata. Used to indicate the currency of the package. If used the two values “SUPERSEDED” or “CURRENT” from the vocabulary is used.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">Status of the metadata. Used to indicate the currency of the package using the terms “SUPERSEDED” or “CURRENT”.</p>
```

Comment:
Indicates the status of the package using a fixed vocabulary.

Reply:
Fixed